src/components/menu/menu.tsx

```diff
@@ -271,9 +271,10 @@ class MenuItem extends Component<any, any> {
   render(props: any): VNode<any> {
     const badgeType: string | null =
       props.data.badgeType && !props.isSelected(props.data) ? BadgeType[props.data.badgeType] : BadgeType[props.data.badgeType + 'Active'];
+      const isAdvanced = props.data.value == "Advanced captions settings" ? true : false;
```

Comment:
is this guaranteed to be set ? to be in english ? 
is there no other way to check this condition except for comparing strings ?

Reply:
Hello, @SivanA-Kaltura ! After checking the props, I didn't find any ID or unique key to get this element. Do you have any suggestions here?
<img width="363" alt="image" src="https://github.com/kaltura/playkit-js-ui/assets/158462211/c021b5f6-d457-47a1-8d1e-9a36fa09337d">

Reply:
Update: I've added a new key to check on it

Reply:
done